source/sites/publicodes/conference/FilterBar.tsx

```diff
@@ -65,15 +85,21 @@ export default ({
 							inputMode="decimal"
 							allowNegative={false}
 						/>{' '}
-						tonnes.
+						tonnes
 					</label>
 				</small>
-				<button
-					title={t('Fermer les options de tri')}
-					onClick={() => setVisible(false)}
-				>
-					{emoji('❌')}
-				</button>
+			</div>
+			<div>
+				<small>
+					<Checkbox
+						name="setRealTimeMode"
+						id="setRealTimeMode"
+						label="Afficher seulement les simulations terminées"
+						showLabel
+						checked={!realTimeMode}
+						onChange={() => setRealTimeMode(!realTimeMode)}
```

Comment:
Je mettrais bien un tracker ici pour voir si l'option est utilisée
```suggestion
						onChange={() => {
						  tracker.push(['trackEvent', 'Mode groupe', realTimeMode ? 'Désactivation du mode temps réel' : 'Activation du mode temps réel' ])
						  setRealTimeMode(!realTimeMode)
						}}
```